index.js

```diff
@@ -11,6 +11,7 @@ export * from './components/combobox';
 export * from './components/collapsible';
 export * from './components/description_list';
 export * from './components/dropdown';
+export * from './components/emoji_picker';
```

Comment:
Shouldn't this be exported from `emoji.js` instead?

Reply:
Hmm not sure what you mean. 
But for now its beign exported same as the vue3 version.

Reply:
There are some files that should be in emoji that are in here already. I have a task on my list to fix this it's been a problem for a while.